data-r2dbc/src/test/groovy/io/micronaut/data/r2dbc/TestResourcesDatabaseTestPropertyProvider.groovy

```diff
@@ -38,7 +38,6 @@ trait TestResourcesDatabaseTestPropertyProvider implements TestPropertyProvider
             case Dialect.ORACLE:
                 return "oracle"
             case Dialect.MYSQL:
-//                return "mariadb"
```

Comment:
Do we need this changes?

Reply:
Just removed commented code, did not think it was needed.